dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -624,27 +673,30 @@ private void search(List<String> args) {
     }
 
     private void convert(final List<String> args) throws ApiException, IOException {
-        if (args.isEmpty() || (containsHelpRequest(args) && !args.contains("cwl2json") && !args.contains("wdl2json") && !args
-                .contains("entry2json"))) {
+        if (args.isEmpty() || (containsHelpRequest(args) && !args.contains(CWL_2_JSON) && !args.contains(WDL_2_JSON) && !args
+                .contains(ENTRY_2_JSON))) {
             convertHelp(); // Display general help
         } else {
             final String cmd = args.remove(0);
             if (null != cmd) {
                 switch (cmd) {
-                case "cwl2json":
+                case CWL_2_JSON:
                     cwl2json(args, true);
                     break;
-                case "cwl2yaml":
+                case CWL_2_YAML:
                     cwl2json(args, false);
                     break;
-                case "wdl2json":
+                case WDL_2_JSON:
                     wdl2json(args);
                     break;
-                case "entry2json":
+                case ENTRY_2_JSON:
                     handleEntry2json(args);
                     break;
                 default:
-                    invalid(cmd);
+                    List<String> possibleCommands = new ArrayList<String>();
```

Comment:
Implements suggestion tool for `dockstore tool convert INVALID_COMMAND` and `dockstore tool convert INVALID_COMMAND`